src/neptune/metadata_containers/metadata_container.py

```diff
@@ -143,6 +147,43 @@ def __init__(
 
         self._startup(debug_mode=mode == Mode.DEBUG)
 
+        if sys.version_info >= (3, 7):
+            try:
+                os.register_at_fork(
+                    after_in_child=self._handle_fork_in_child, after_in_parent=self._handle_fork_in_parent
+                )
+            except AttributeError:
+                pass
+
+    """
+    OpenSSL's internal random number generator does not properly handle forked processes.
```

Comment:
Let's move the comment and `ssl.RAND_bytes(100)` to a separate utility function.

Reply:
Done.